devDocs/userGuideStandards.md

```diff
@@ -0,0 +1,28 @@
+# User Guide Standards
+This document aims to create a standard style guide for writing documentation in the User Guide.
+
+The principles outlined in ["The Documentation System" guide for reference materials](https://documentation.divio.com/reference/) are encouraged when working on the User Guide and this document.
+
+## General standards
+- Key commands should be lowerCamelCase and encapsulated in monospace code-block formatting.
+  - e.g. ` ``NVDA+control+upArrow`` `
+- When referring to Windows terminology, follow the [Windows style guide](https://docs.microsoft.com/en-us/style-guide/welcome/).
+  - For instance, instead of "system tray" refer to "notification area"
+
+## Feature flags

```

Comment:
I don't think this is specific to feature flags (the configSpec concept), this risks alluding to that.
Perhaps:
```suggestion
## Feature settings

```